app/views/home/businesses.html.erb

```diff
@@ -21,49 +21,49 @@
 </div>
 
 <div class="container-lg py-6 py-3 px-3 text-center">
-  <h2 class="alt-h2">Your business depends on it</h2>
+  <h2 class="alt-h2"><%=t "Your business depends on it" %></h2>
 
   <p class="alt-lead text-gray mx-auto text-center mb-6">
-    Companies need open source software to run effectively.
+  <%=t "Companies need open source software to run effectively" %>
   </p>
 </div>
 <div class="container-lg mx-auto gut-lg text-center">
   <div class="mb-6 col-md-6 float-md-left">
     <%= image_tag("improve-technology.svg", class: "img-responsive mb-3", height: "100px") %>
-    <h3 class="alt-h3 mb-2">Improve the technology you rely on</h3>
+    <h3 class="alt-h3 mb-2"><%=t "Improve the technology you rely on" %></h3>
     <p class="text-gray">
-      Direct participation in open source gives your organization a voice in the software you depend on.
+      <%=t "Direct participation in open source" %>
```

Comment:
Ok. I think it'd be good to use the full length one to avoid confusion and have the HTML be more readable for direct edits for people with less localisation experience. Relatedly, this just made me think it would be good to have some sort of test checking that people don't edit strings here and break localisation on the site. Also: what do you think should happen if someone e.g. wants to change an English string? Keep the old localised string or default to the English one until it's been updated? I have no idea what best practise is here and would love to learn from you.